lib/team_api/team_member_schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "team": {
+    "title": "team - [team.member].yml files",
+    "description": "Team member metadata",
+    "$schema": "http://json-schema.org/draft-04/hyper-schema",
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "Unique name for the team member, derived from their GSA email address. Must match the file basename."
+      },
+      "deprecated_name": {
+        "type": "string",
+        "description": "Previous value for the `name` field (and the previous file name)"
+      },
+      "full_name": {
+        "type": "string",
+        "description": "Full proper name of the team member"
+      },
+      "first_name": {
+        "type": "string",
+        "description": "Team member's first name"
+      },
+      "last_name": {
+        "type": "string",
+        "description": "Team member's last name"
+      },
+      "github": {
+        "type": "string",
+        "description": "Team member's GitHub username"
+      },
+      "email": {
+        "type": "string",
+        "description": "Team member's GSA email address"
+      },
+      "location": {
+        "type": "string",
+        "description": "Team member's location"
+      },
+      "slack": {
+        "type": "string",
+        "description": "Team member's Slack username"
+      },
+      "resume": {
```

Comment:
I'm doing this all based on what I've observed. If we want to change the field names, I recommend doing that in separate Issues/PRs.